### ISSUE TITLE
Carousell Buttons with no tags

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#586]
https://swiss.github.io/styleguide/de/teaser.html#05-teaser-05-carousel

![image](https://user-images.githubusercontent.com/7510842/32599936-46723bb0-c53e-11e7-818e-f8145cc74384.png)

**Comment by gillerr** [comment]
Focus and focus vertical are displayed as carousel on mobile. In this case, labels are missing on the "previous" and "next" buttons

**Comment by bit-pro-iew-eui** [comment]
Ok, validated
